sdk/webpubsub/azure-messaging-webpubsubservice/setup.py

```diff
@@ -61,9 +61,9 @@
         ]
     ),
     install_requires=[
-        "azure-core<2.0.0,>=1.10.0",
-        "msrest>=0.6.18",
-        "cryptography>=2.1.4",
+        "azure-core<2.0.0,>=1.16.0",
+        "msrest>=0.6.21",
+        "cryptography>=2.8.0",
```

Comment:
qq: why is the cryptography version bumped by so much?

Reply:
to pass CI